packages/charts/react-charting/src/components/DonutChart/DonutChart.base.tsx

```diff
@@ -265,11 +263,17 @@ export class DonutChartBase extends React.Component<IDonutChartProps, IDonutChar
         focusZonePropsInHoverCard={this.props.focusZonePropsForLegendsInHoverCard}
         overflowText={this.props.legendsOverflowText}
         {...this.props.legendProps}
+        canSelectMultipleLegends={this.props.canSelectMultipleLegends}
+        onChange={this._onLegendChange}
```

Comment:
>onChange={this._onLegendChange} [](http://example.com/codeflow?start=8&length=31)

keep using the action event handler. As this can be overridden for a specific legend as per the caller.

Dont modify existing behavior.

Reply:
modified as discussed offline